batcher/aligned-batcher/src/eth/mod.rs

```diff
@@ -19,15 +15,31 @@ pub struct BatchVerified {
     pub batch_merkle_root: [u8; 32],
 }
 
-pub type BatcherPaymentService =
-    BatcherPaymentServiceContract<SignerMiddleware<Provider<Http>, Wallet<SigningKey>>>;
+pub type BatcherPaymentService<T> = BatcherPaymentServiceContract<
```

Comment:
do we need it to be generic over T? The batcher payment service only seems to use `Http` inside the provider generic

Reply:
Changed on 5f8cf3ab09970f06aa30c644cef65333999c5f33